src/util/util.c

```diff
@@ -589,6 +589,14 @@ errno_t sss_filter_sanitize_ex(TALLOC_CTX *mem_ctx,
             output[j++] = '5';
             output[j++] = 'c';
             break;
+        case '\n':
+            output[j++] = '\\';
+            output[j++] = '0';
+            output[j++] = 'd';
+            output[j++] = '\\';
```

Comment:
0x0d is '\r'.
Should we split '\n' and '\r' into two cases?

Or could you explain why they are together?